SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/TimeToLive.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.AppCenter.Data
+{
+    /// <summary>
+    /// Constants for document time to live on device.
+    /// </summary>
+    public static class TimeToLive
```

Comment:
I know that native android uses static class with `static final` fields for Infinite/NoCache/Default and it static class works fine in C#. 

Note one subtle difference between Java/Objective-C and C# implementation is that the former uses `long` or `NSTimeInterval` but the latter uses `TimeSpan`.

Have we considered using enum here? That is the more idiomatic way of dealing with enum values in C# with the benefits of better type check. I don't have a strong opinion on this, just want to bring up the question.

@guperrot @achocron @MatkovIvan What's your thoughts on this?

Reply:
It has customizable value, so enum isn't an option here

Reply:
These are just the default values that we use, and `NoCache` and `Infinite` basically have special meanings.  We need to be able to use any time span though, so I think this implementation is fine @dhei